preupg/xmlgen/xml_utils.py

```diff
@@ -209,16 +207,17 @@ def check_script_modification(self, key, k):
         self.update_values_list(self.rule, "{" + k + "}", key[k])
 
     def prepare_sections(self):
-        """The function prepares all tags needed for generation group.xml file."""
-        for main, self.keys in self.ini_files.items():
-            if main.endswith("group.ini"):
+        """The function prepares all tags needed for generation group.xml file.
+        """
+        for ini_filepath, ini_file_content in iter(self.ini_files.items()):
```

Comment:
Isn't the `iter()` call redundant?  (Just checked with python 2.7 and it seems to behave the same)